nautobot/core/admin.py

```diff
@@ -60,3 +57,49 @@ class NautobotModelAdmin(ModelAdmin):
         models.ForeignKey: {"widget": widgets.StaticSelect2},
         models.ManyToManyField: {"widget": widgets.StaticSelect2Multiple},
     }
+
+
+class LogEntryAdmin(NautobotModelAdmin):
+    """ModelAdmin for Django admin log entries that is read-only."""
+
+    date_hierarchy = "action_time"
+    search_fields = ["change_message", "object_repr", "user__username"]
+    list_display = [
+        "action_time",
+        "user",
+        "content_type",
+        "object_link",
+        "action_flag",
+    ]
+
+    def has_add_permission(self, request):
+        """Disable add permission."""
+        return False
+
+    def has_change_permission(self, request, obj=None):
+        """Disable change permission."""
+        return False
+
+    def has_delete_permission(self, request, obj=None):
+        """Disable delete permission."""
+        return False
+
+    def has_view_permission(self, request, obj=None):
+        return request.user.is_staff
```

Comment:
Something seems wrong/missing here - I created a non-superuser `is_staff` user and they don't have a `Log Entries` section on their admin home page:

![image](https://github.com/nautobot/nautobot/assets/5603551/895cc381-d65c-4e0f-95a3-237882b841cb)

Although they do appear to have access to http://localhost:8080/admin/admin/logentry/.

This is different behavior than I see for non-superuser staff users for other admin pages - for `groups`, `tokens`, and `permissions` the home page items only appear, and the URL is only accessible, if the user explicitly has object permissions for those content-types.

Reply:
Yeah this should just not be overloaded and allowed to use the default permissions controls in the `ModelAdmin`. My attempt to gate by `is_staff` appears to have been flawed.